### ISSUE TITLE
bug: pnpm run dev fails

### ISSUE THREAD

**Issue by sanjeethboddi** [t3-oss/create-t3-turbo#1135]
### Provide environment information

  System:
    OS: macOS 14.5
    CPU: (8) arm64 Apple M2
    Memory: 73.63 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.16.0 - ~/.nvm/versions/node/v20.16.0/bin/node
    Yarn: 1.22.21 - ~/.nvm/versions/node/v18.12.1/bin/yarn
    npm: 10.8.1 - ~/.nvm/versions/node/v20.16.0/bin/npm
    pnpm: 9.5.0 - /opt/homebrew/bin/pnpm

### Describe the bug

pnpm run  dev:next fails

```
> create-t3-turbo@ dev:next /sd/x
> turbo watch dev -F @acme/nextjs...

turbo 2.0.11

• Packages in scope: @acme/api, @acme/auth, @acme/db, @acme/eslint-config, @acme/nextjs, @acme/prettier-config, @acme/tailwind-config, @acme/tsconfig, @acme/ui, @acme/validators
• Running dev in 10 packages
• Remote caching disabled
  × failed to connect to daemon
  ╰─▶ server is unavailable: channel closed

 ELIFECYCLE  Command failed with exit code 1.
```

### Link to reproduction

https://github.com/t3-oss/create-t3-turbo

### To reproduce

just clone and run

**Comment by sanjeethboddi** [comment]
<img width="1167" alt="image" src="https://github.com/user-attachments/assets/e6936cd2-1650-47e7-8f8c-6d04c4c213b4">

**Comment by juliusmarminge** [comment]
Try `turbo daemon restart`

**Comment by dBianchii** [comment]
I reported this same issue here:

https://github.com/vercel/turbo/issues/8491

For now, I need to run `turbo daemon clean` every once in a while and try running the command again 2 times

**Comment by sanjeethboddi** [comment]
> I reported this same issue here:
> 
> [vercel/turbo#8491](https://github.com/vercel/turbo/issues/8491)
> 
> For now, I need to run `turbo daemon clean` every once in a while and try running the command again 2 times

After doing `turbo daemon clean` I'm getting this error `discovery failed: bad grpc status code: The operation was cancelled`

will track the status of [vercel/turbo/issues/8491](https://github.com/vercel/turbo/issues/8491)

<img width="1169" alt="image" src="https://github.com/user-attachments/assets/5203bcab-9e6f-4bbd-adc8-b3c2f3aae1f8">

**Comment by sanjeethboddi** [comment]
@dBianchii Could you please suggest other ways to get the template up and running without using turbo?

**Comment by dBianchii** [comment]
> @dBianchii Could you please suggest other ways to get the template up and running without using turbo?

I only use it with turbo. You can also go to apps/NextJS and run `pnpm dev` which should also work

**Comment by juliusmarminge** [comment]
Would you please mind checking if upgrading to latest turbo (part of #1144) fixes your issue?

Run `pnpm turbo daemon restart` too after upgrading for good measure

**Comment by sanjeethboddi** [comment]
@juliusmarminge No it doesn't fix the issue.

**Comment by dBianchii** [comment]
Has anyone found a solution? It's so annoying having to restart the daemon every time

**Comment by Shashivadan** [comment]
NO. I'm also encountering the same issue

**Comment by techwithanirudh** [comment]
Yeah, I'm having the same issue here while using t3-turbo. I'm on the latest version of Turbo.

**Comment by parteekmalik** [comment]
is someone found a solution for this?
